src/generator/helpers.ts

```diff
@@ -164,13 +167,17 @@ export function generateParameterInfo(op: Operation): ParamInfo[] {
     }
     // include client and method params
     const global = param.implementation === ImplementationLocation.Client;
-    params.push(new paramInfo(param.language.go!.name, param.schema.language.go!.name, global, param.required === true));
+    let isHost = false;
+    if (global) {
+      isHost = (param.extensions!['x-ms-priority'] === 0 && param.extensions!['x-in'] === 'path');
```

Comment:
Is the `!` going to be problematic?  `extensions` can legitimately be undefined.

Reply:
I can add an if to check if it's undefined before

Reply:
if (global && param.extensions !== undefined)

Reply:
If you use `?` instead it gives you the same effect.

Reply:
ok 👍